tests/ui/conftest.py

```diff
@@ -95,29 +297,29 @@ def user(create_superuser, my_base_url, fxa_account, jwt_token):
 @pytest.fixture(scope='function')
 def live_server(request, transactional_db):
```

Comment:
Well each ```live_server``` has its own database. If it's session scoped the fixtures that write to the db would get a duplicate key error.